source/release-notes/1.2.txt

```diff
@@ -0,0 +1,46 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.2
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.2 Released February 15, 2023
+
+Support for Older Versions of MongoDB
+-------------------------------------
+
+{+c2c-product-name+} 1.2 adds support for sync between clusters running 
+older versions of MongoDB.
+
+Sync between MongoDB 5.0, 4.4, and 4.2 clusters replicate CRUD operations only.  
+
+``mongosync`` **cannot** detect DDL operations in pre-6.0 clusters.
+DDL operations, such as creating and dropping databases and collections,
+adding indexes, etc. during replication can cause the destination cluster
+to fall out of sync with the source.
+
+For more information, contact your account team or 
+`MongoDB <https://www.mongodb.com/contact>`__.
+
+
+Guardrails for Unsupported Versions of MongoDB
+----------------------------------------------
+
+``mongosync`` now returns an error if the user calls :ref:`c2c-api-start`
```

Comment:
[suggestion]

Consider "returns an error if you call...".

Reply:
Fixed